src/agent/README.md

```diff
@@ -1,6 +1,6 @@
 # Kata Agent in Rust
 
-This is a rust version of the [`kata-agent`](https://github.com/kata-containers/agent).
+This is a rust version of the [`kata-agent`](../agent).
```

Comment:
Oh, it is intentionally referring to the go version of `kata-agent` since this is already the rust version.

Reply:
oh, sorry for this mistake. thanks @bergwolf